app/code/Magento/Cms/view/adminhtml/web/js/folder-tree.js

```diff
@@ -101,8 +104,45 @@ define([
                             tree.jstree('select_node', folderEl);
                         });
                     }
+                },
+
+                /**
+                 * Get currentPath decode it returns new path array
+                 */
+                _parseCurrentPath = function () {
+                    var paths = [];
+
+                    $.each(encodedPath, function (i, val) {
+                        var isLastElement = i === encodedPath.length - 1;
+
+                        if (isLastElement) {
+                            paths[i] = window.MediabrowserUtility.pathId.replace(',,', '--');
+                        } else {
+                            paths[i] = Base64.encode(val).replace('==', '--');
+                        }
+                    });
+                    paths.unshift('root');
+                    paths.reverse();
+
+                    return paths;
                 };
 
+            $(window).on('reload.MediaGallery', function () {
+                encodedPath = Base64.decode(window.MediabrowserUtility.pathId.replace(/--|,,/, '==')).split('/');
+                pathId =  window.MediabrowserUtility.pathId.replace(',,', '--');
+
+                tree.jstree('deselect_all');
+
+                if (encodedPath.length > 1) {
+                    path =  _parseCurrentPath(encodedPath);
```

Comment:
We are actually using it here as a parameter. Highlighting to support the comment above.